packages/graphql-function-transformer/src/lambdaArns.ts

```diff
@@ -0,0 +1,23 @@
+import { Fn, Refs } from 'cloudform-types';
+import { ResourceConstants } from 'graphql-transformer-common';
+
+export function lambdaArnResource(name: string, region?: string) {
+    const substitutions = {};
+    if (referencesEnv(name)) {
+        substitutions['env'] = Fn.Ref(ResourceConstants.PARAMETERS.Env)
+    }
+    return Fn.Sub(
+        lambdaArnKey(name, region),
```

Comment:
Shouldn't special handling be added here when env = NONE similar to existing multi-env cloudformation templates? The NONE env is legacy environment created back when multi-env was not supported.